X584.bpr

```diff
@@ -83,14 +83,14 @@ CodePage=1251
 
 [Version Info Keys]
 CompanyName=
-FileDescription=
-FileVersion=1.0.0.0
-InternalName=
-LegalCopyright=
+FileDescription=Симулятор процессора К584ВМ1
+FileVersion=1.30.0.0
+InternalName=X584
+LegalCopyright=(c) X584 developers, 2005-2024
 LegalTrademarks=
-OriginalFilename=
-ProductName=
-ProductVersion=1.0.0.0
+OriginalFilename=X584.exe
+ProductName=X584
+ProductVersion=1.30
```

Comment:
It's better to use the same value for both `ProductVersion` and `FileVersion`, i.e. `1.30.0.0`